Bottom: ab049e02d4691d912e69b8417f5caee161347f40
Top:    b2578af105ba6c38c45331d6fc64dcee25c0fdea
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-01-30 14:30:15 +0530

Refresh of sched-fair-paraphrase

---

diff --git a/kernel/sched/core.c b/kernel/sched/core.c
index fc1dfc007604..30704f2037db 100644
--- a/kernel/sched/core.c
+++ b/kernel/sched/core.c
@@ -4642,23 +4642,6 @@ int idle_cpu(int cpu)
 	return 1;
 }
 
-/**
- * available_idle_cpu - is a given CPU idle for enqueuing work.
- * @cpu: the CPU in question.
- *
- * Return: 1 if the CPU is currently idle. 0 otherwise.
- */
-int available_idle_cpu(int cpu)
-{
-	if (!idle_cpu(cpu))
-		return 0;
-
-	if (vcpu_is_preempted(cpu))
-		return 0;
-
-	return 1;
-}
-
 /**
  * idle_task - return the idle task for a given CPU.
  * @cpu: the processor in question.
